src/spatial_ref/srs.rs

```diff
@@ -33,6 +175,24 @@ impl CoordTransform {
         })
     }
 
+    pub fn new_ex(
+        sp_ref1: &SpatialRef,
```

Comment:
Maybe something like

```suggestion
        source: &SpatialRef,
```

(and `target` below)

Reply:
Good call. Done.